Sources/SmokeFrameworkCodeGeneration/SmokeFrameworkCodeGeneration.swift

```diff
@@ -250,6 +268,34 @@ extension ServiceModelCodeGenerator {
                 generateStreamlinedOperationsContextProtocolGenerator(generationType: generationType,
                                                                       asyncInitialization: asyncInitialization)
             }
+        } else if generationType.isWithPlugin {
+            generateCodeGenDummyFile(forPackagePostfix: "OperationsHTTP1",
+                                     plugin: "SmokeFrameworkGenerateHttp1",
+                                     generationType: generationType)
         }
     }
+
+    func generateCodeGenDummyFile(forPackagePostfix packagePostix: String,
+                                  plugin: String,
+                                  generationType: GenerationType) {
+        let fileBuilder = FileBuilder()
+        let baseFilePath = applicationDescription.baseFilePath
+        let fileName = "codegen.swift"
+        let filePath = "\(baseFilePath)/Sources/\(applicationDescription.baseName)\(packagePostix)"
+        
+        if generationType.isUpdate {
+            guard !FileManager.default.fileExists(atPath: "\(filePath)/\(fileName)") else {
+                return
+            }
+        }
+        
+        fileBuilder.appendLine("""
+            //
+            //  This package is code generated by the smoke-framework-application-generate \(plugin) plugin.
```

Comment:
It could be a good idea to clarify this further, even if the readme goes over it.

"Due to a current limitation of the SPM plugins for code generators, a placeholder Swift file is required in each package to avoid the package as being seen as empty. These files need to be a Swift file but doesn't require any particular contents."